app/models/user.rb

```diff
@@ -591,6 +595,10 @@ def cohorts
     portal_teacher ? portal_teacher.cohorts : (portal_student ? portal_student.cohorts : [])
   end
 
+  def cohort_projects
+    cohorts.map {|c| c.project}.flatten.compact.uniq
```

Comment:
Fixed in: https://github.com/concord-consortium/rigse/commit/b0f3c2d96c27fb574f4493dbbee4cada51b7e8f3